ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -58,15 +57,17 @@
                 break;
             case 'custom':
                 var oldType = this.options.queryModel.get('type');
-                this.options.queryModel.set({
-                    type: 'custom',
-                    title: this.model.get('name')
-                });
+
                 user.getQuerySettings().set({
                     type: 'custom',
                     template: this.model.toJSON()
                 });
 
+                this.options.queryModel.set({
```

Comment:
Did the old order cause issues (maybe eventing problems)?

Reply:
Yeah, the user preferences were accessed before they were set, causing the first click to fail.